mlrun/utils/notifications/notification/base.py

```diff
@@ -59,6 +69,13 @@ def load_notification(
     ) -> None:
         self.params = params or {}
 
+    @classmethod
+    def enrich_default_params(cls, params: dict, default_params: dict = None) -> dict:
```

Comment:
```suggestion
    def _enrich(cls, params: dict, default_params: dict = None) -> dict:
```
do you call it externally at all?

Reply:
See above